### PR TITLE
New definition for vehicle turnaround times

### PR DESCRIPTION
**Pull request by joakimnordling**
This PR adds a new `Transport/Vehicle/TurnaroundTimes_v0.1` definition that should eventually replace the old, now deprecated, 
`Transport/Vehicles/TurnaroundTime/VehicleTurnaroundTime_v0.1` definition. The path and name is improved and the actual definition improved to better match real world use cases.

### CODE REVIEW

File: src/Transport/Vehicle/TurnaroundTimes_v0.1.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        title="Location identifier",
+        max_length=40,
+        description="Location identification number based on UN/LOCODE, or other "
+        "similar identification number of the transport location or facility.",
+        examples=["FI OUL"],
+    )
+    start_time: datetime.datetime = Field(
+        ...,
+        title="Start time",
+        description="The start time of the performance period, in RFC 3339 format.",
```
Comment (lietu) [inline src/Transport/Vehicle/TurnaroundTimes_v0.1.py:60]: What performance period?

File: src/Transport/Vehicle/TurnaroundTimes_v0.1.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        max_length=40,
+        examples=["ABC-123"],
+    )
+    vehicle_type: Optional[str] = Field(
+        None,
+        title="Vehicle type",
+        description="The type of the vehicle.",
+        max_length=40,
+        examples=["truck"],
+    )
```
Comment (lietu) [inline src/Transport/Vehicle/TurnaroundTimes_v0.1.py:23]: Could we not get some enum for this?

File: src/Transport/Vehicle/TurnaroundTimes_v0.1.py
Comment (lietu) [inline src/Transport/Vehicle/TurnaroundTimes_v0.1.py:1]: Why is this path different? What is the difference between `Vehicle` and `Vehicles` that is being implied here?

File: src/Transport/Vehicle/TurnaroundTimes_v0.1.py (hunk 2 above)
Comment (joakimnordling) [reply src/Transport/Vehicle/TurnaroundTimes_v0.1.py:23]: I checked, not at this stage. There's a note in our internal material for this field saying `Should be enum in the future when all values are known`.

File: src/Transport/Vehicle/TurnaroundTimes_v0.1.py
Comment (joakimnordling) [reply src/Transport/Vehicle/TurnaroundTimes_v0.1.py:1]: All other paths are singular. We have for example `CargoHandlingEquipment`, `Battery`,  `Market` etc. The original `Vehicles` was an outlier to this pattern that was just not detected in time. The original path also had `../Vehicle/.../VehicleTurnaroundTime`, so a bit of duplication of `Vehicle` that is being fixed now.

File: src/Transport/Vehicle/TurnaroundTimes_v0.1.py (hunk 1 above)
Comment (joakimnordling) [reply src/Transport/Vehicle/TurnaroundTimes_v0.1.py]: Good point, updated.

**Review by lietu — APPROVED** [review]
Awesome, thanks

**Pull request merged**